tests/test_decode_tokens.py

```diff
@@ -38,9 +38,10 @@ def app():
 @pytest.fixture(scope="function")
 def default_access_token(app):
     with app.test_request_context():
+        identity_claim_key = config.identity_claim_keys[0]
```

Comment:
We could encode all the different identities into every token, but since we decided we are migrating over from `identity` to `sub` I figured we could give priority to the first one (which will be `sub`) then when we remove the fork eventually it will be like the fork never modified things, instead of going from `identity` -> `sub` and `identity` -> `sub`